README.md

```diff
@@ -44,6 +44,61 @@ Include the download centre nav-menu:
     ...
 ```
 
+## WS mode
+Package can be used with websockets to provide asynchronous communication
+between frontend and backend.
+If package will be used in websockets mode additional settings must be applied.
+* Inside common settings `WS_MODE` must be toggled on, and master url need to be provided.
+    ```
+    ASYNC_DOWNLOADS_WS_MODE = True
+    ASYNC_DOWNLOADS_MASTER_URL = 'http://example.com'
```

Comment:
Do we need a `MASTER_URL`? Is it not the same as `SITE_URL`?

Reply:
That would be good name for future usage with multiple apps? We need to pass there url for our 'master' app to provide information for websocket to connect. That would be not intuitive in future I think, because in settings you would have `SITE_URL=current.app` and also `ASYNC_DOWNLOADS_SITE_URL=master.app`. But I do not know what would be better naming, maybe just `WS_URL` and add readme that it need to be site url for single app usage?

Reply:
Can be done in that way but it will require also to configure each app as ASGI and create endpoints there too. But for this lib  alone in mind I will change that to 'SITE_URL': 9270780